prow/cluster/components/tide_deployment.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
is `starter_merge.yaml` checked in by mistake?

Reply:
Indeed, removing it.